### PR TITLE
Filesystem doc

### PR DESCRIPTION
**Pull request by cgwalters**
filesystem: Doc usr/etc

Came up in review.

Signed-off-by: Colin Walters <walters@verbum.org>

---

filesystem: Doc etc.transient

I noticed this was missing.

Signed-off-by: Colin Walters <walters@verbum.org>

---

doc: Expand on prepare-root

Per review.

Signed-off-by: Colin Walters <walters@verbum.org>

---

### CODE REVIEW

File: docs/src/filesystem.md [hunk 1]
```diff
@@ -95,6 +95,13 @@ would require external intervention to apply.
 
 For more on configuration file best practices, see [Building](building/guidance.md).
 
+### `/usr/etc`
+
+The `/usr/etc` tree is generated client side and contains the defaults. This should generally
```
Comment (vrothberg) [inline docs/src/filesystem.md:101]: Can you specify which defaults (e.g., for which tools)?

File: docs/src/filesystem.md [hunk 2]
```diff
@@ -190,6 +191,24 @@ write (transiently, i.e. until the next reboot) to all top-level directories,
...
+not need to use it for any per-machine state, then enabling a transient
+`/etc` is a great way to reduce the amount of possible state drift. Set
+the
+
+```toml
+[etc]
+transient = true
+```
+
+option in `prepare-root.conf`.
```
Comment (vrothberg) [inline docs/src/filesystem.md:208]: Specifying the full path of `/usr/lib/ostree/prepare-root.conf` may prevent readers from searching.  I did a CTRL+f as I forgot the path.

**Review by vrothberg — COMMENTED** [review]
Thanks!
Just two tiny nits

File: docs/src/filesystem.md [hunk 3]
```diff
@@ -95,6 +95,13 @@ would require external intervention to apply.
 
 For more on configuration file best practices, see [Building](building/guidance.md).
 
+### `/usr/etc`
+
+The `/usr/etc` tree is generated client side and contains the default container image's
+view of `/etc`. This should generally be considered an internal implmentation detail
```
Comment (vrothberg) [inline docs/src/filesystem.md:102]: s/implmentation/implementation/

**Review by vrothberg — COMMENTED** [review]
LGTM, thanks!
Just one typo. Feel free to merge after.

File: docs/src/filesystem.md (hunk 3 above)
Comment (cgwalters) [reply docs/src/filesystem.md]: Turns out there were 3 of those...

**Comment by cgwalters** [comment]
> Feel free to merge after.

At a technical level today this repo requires an official GH approval to merge

**Review by jeckersb — APPROVED** [review]
I put on my @vrothberg disguise and approve :laughing:

**Comment by vrothberg** [comment]
Thanks, @jeckersb! My calendar is a mess this week.

**Pull request merged**